cmd/root.go

```diff
@@ -99,7 +99,7 @@ func init() {
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
 
-	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.spinup-cli spinup.yaml)")
+	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.spinup.yaml)")
```

Comment:
Below in `initConfig` you have:
`Search config in home directory with name ".spinup" (without extension)`

Reply:
thats correct, cobra supports `yaml`, `yml`, `json`... maybe others